README.md

```diff
@@ -1 +1,12 @@
 # hw01
+
+## My introduction
+
+My name is **Soaham Bharti**. I am a first year PhD student in behavioral marketing at the *Booth School of Business*.
+
+Thing I want to study:
+
+* How consumers and other decision makers navigate uncertainty
```

Comment:
Try different combinations of lists (either unordered and ordered list or two different levels of unordered lists) next time.